docs/release-notes/8.10.asciidoc

```diff
@@ -1,6 +1,17 @@
 [[release-notes-header-8.10.1]]
 == 8.10
 
+[discrete]
+[[release-notes-8.10.4]]
+=== 8.10.4
+
+[discrete]
+[[bug-fixes-8.10.4]]
+==== Bug fixes
+* Fixes a bug in Timeline that prevented the **Show top _x_** action from showing results ({pull}168339[#168339]).
+* Fixes a bug in Timeline that prevented the **Show top _x_** action from using the Timeline query ({pull}165109[#165109]).
```

Comment:
Hey Natasha!

PR #165109 looks duplicated. It is present on 8.10.3 (line 12) and 8.10.4 (line 31).

Reply:
There is another `Top N` fix on 8.10.14: https://github.com/elastic/kibana/pull/168339

Reply:
Hey, @machadoum! RE your question about #165109: Yes, this is correct. The update I made to the 8.10.3 release notes was to incorporate your feedback [here](https://github.com/elastic/security-docs/pull/4011#discussion_r1351726553). 

RE your second comment about https://github.com/elastic/kibana/pull/168339: The bug fix is doc'd on [line 12](https://github.com/elastic/security-docs/pull/4024/files#diff-c080ead95d6cded24b73bf926af3bb9229d0b0687959c335947ea08a42ca4c73R12). Let me know if it needs any changes :)

Reply:
Confirmed that https://github.com/elastic/kibana/pull/165109 should only be doc'd in 8.10.3. 

```suggestion
```